tests/presets.spec.js

```diff
@@ -740,6 +740,289 @@ describe('Presets', function() {
 
     });
 
+    describe('.cargo', function() {
+
+      describe('given Cargo.toml does not exist', function() {
+
+        beforeEach(function() {
+          this.cwd = tmp.dirSync();
+        });
+
+        afterEach(function() {
+          this.cwd.removeCallback();
+        });
+
+        it('should yield an error', function(done) {
+          presets.updateVersion.cargo({}, this.cwd.name, '1.0.0', (error) => {
+            m.chai.expect(error).to.be.an.instanceof(Error);
```

Comment:
Now that there's errors being emitted from several places in the code, is it also worth checking the error-message in these unit-tests to check that the right error is being thrown by the right test? \*shrug\*

Reply:
Maybe there is an error-type kind of thing that can be used - I will check and let you know.

Reply:
`Error` has a `name` property that looks like the natural way to determine error types. Another way is to use custom error types, but that is too much for this case I think.

Reply:
Ahh, if there's no easy way to get the actual error-message during the unit-tests, then I'm happy with what you've got currently.

Reply:
Hm, maybe we can check the actual message with string formatting only...

Reply:
String formatting worked..